_data/projects/emby.yml

```diff
@@ -0,0 +1,15 @@
+name: Emby
+desc: The open media solution for your personal videos, music, photos, and Live TV.
+site: https://github.com/MediaBrowser/Emby
+tags:
+- ".NET"
+- javascript
+- C#
+- IOS
+- Android
+- Windows Phone
+- Windows 10
+- Roku
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/MediaBrowser/Emby/labels/up%20for%20grabs
```

Comment:
I've made the correction, thank you.

On Thu, Jul 23, 2015 at 6:47 PM, Brendan Forster <notifications@github.com>
wrote:

> In _data/projects/emby.yml
> <https://github.com/dahlbyk/up-for-grabs.net/pull/205#discussion_r35380612>
> :
>
> > @@ -0,0 +1,15 @@
> > +name: Emby
> > +desc: The open media solution for your personal videos, music, photos, and Live TV.
> > +site: https://github.com/MediaBrowser/Emby
> > +tags:
> > +- ".NET"
> > +- javascript
> > +- C#
> > +- IOS
> > +- Android
> > +- Windows Phone
> > +- Windows 10
> > +- Roku
> > +upforgrabs:
> > +  name: up-for-grabs
> > +  link: https://github.com/MediaBrowser/Emby/labels/up%20for%20grabs
>
> I think this link should be
> https://github.com/MediaBrowser/Emby/labels/up-for-grabs - the %20s are
> turned into spaces, which means you don't get the right label.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dahlbyk/up-for-grabs.net/pull/205/files#r35380612>.
>



-- 
Luke Pulverenti